lib/rules/rules.json

```diff
@@ -21,7 +21,8 @@
     },
     "last": {
         "compatible": true,
-        "alternative": "Array.prototype.slice()"
+        "alternative": "Array.prototype.at(-1) or Array.prototype.slice()",
```

Comment:
Array.prototype.slice() may not be a relevant alternative anymore depending on the version of lodash used, as versions greater than 3.10.1 do not support a second argument in .first() and .last() which I suppose is the primary use case for Array.prototype.slice()